arm-network/2016-09-01/swagger/network.json

```diff
@@ -6469,6 +6612,79 @@
         }
       }
     },
+    "BgpPeerStatus": {
```

Comment:
This model is used in responses only (right?) so there is no point for users to modify the properties. Mark them as `readOnly` then.